### DISCUSSION TITLE
Plugin vs Using Dropbox Directly?

### DISCUSSION THREAD

**Discussion by ryanwwest** [remotely-save/remotely-save#106] [Q&A] [answered]
Hi, this looks like a neat plugin for aggregating various different sync options into a nice package. However, I'm wondering if it has any benefits over just putting my Obsidian vault in my Dropbox directly? I currently do this and it works well on PC/Mac aside from some slightly annoying file copies that are saved if you edit the same file on multiple devices too close to each other. On Android, I have to use the Dropsync app to truly store my vault on my device with 2-way sync (the official Dropbox app no longer allows direct file access-stupid), so if this can get around that I could see the benefit there.

**Comment by fyears** [comment, marked as answer]
I developed it basically because I have an iOS device and unfortunately Obsidian cannot work with Dropbox together on iOS.

Moreover the plugin has extra benefits like end to end encryptions.
